tests/data/test-data.sql

```diff
@@ -229,6 +229,7 @@ INSERT INTO `coop_map` (`type`, `name`, `description`, `version`, `filename`) VA
 insert into game_featuredMods (id, gamemod, name, description, publish, git_url, git_branch, file_extension, allow_override) values
   (1, 'faf', 'FAF', 'Forged Alliance Forever', 1, 'https://github.com/FAForever/fa.git', 'deploy/faf', 'nx2', FALSE),
   (6, 'ladder1v1', 'FAF', 'Ladder games', 1, 'https://github.com/FAForever/fa.git', 'deploy/faf', 'nx2', TRUE),
+  (24, 'gw', '', 'Galactic War', 1, NULL, NULL, NULL, NULL),
```

Comment:
`publish` is set to false for GW because you're not supposed to be able to start GW games from the "Host Game" menu. It doesn't mean that the mod is disabled or anything. I think it just affects whether the mod is returned from the API.